frame/tips/src/lib.rs

```diff
@@ -70,8 +68,10 @@ use frame_support::traits::{
 use sp_runtime::{ Percent, RuntimeDebug, traits::{
 	Zero, AccountIdConversion, Hash, BadOrigin
 }};
-
-use frame_support::traits::{Contains, ContainsLengthBound};
+use frame_support::traits::{
+	Contains, ContainsLengthBound,
+	OnUnbalanced, EnsureOrigin,
+};
```

Comment:
```suggestion
use frame_support::traits::{Contains, ContainsLengthBound, OnUnbalanced, EnsureOrigin};
```

(I mostly don't add review comments about formatting but that was a bit too much :D)